<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="findmodel_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="model_le">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>model file</string>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>annotation dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="annot_le">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="annot_find_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>images dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imagesdir_le">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>270</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="imagesdir_find_btn">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>30</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>epoch</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="epoch_le">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>30</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>session ckpt dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="session_ckpt_dir_le">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="session_ckptdir_find_btn">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>summary save dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="summarydir_le">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>140</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="summarydir_find_btn">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>140</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>previous ckpt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="previousckpt_le">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="previousckpt_find_btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QComboBox" name="prevckptnum_dropdown">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>select ckpt number</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setup_new_session_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>setup new session</string>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_6">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>train data dir</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="traindir_le">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="traindir_find_btn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_9">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>new session title</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newsessiontitle_le">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="checkimglabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_10">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>save number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="save_num_le">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>70</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_11">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>gpu usage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="load_ckpt_num_le">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>220</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="annotation_dir_label_12">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>load ckpt num</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gpu_usage_le">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>270</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="gencmd_btn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>320</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>generate cmd</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="gencmd_te">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>481</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
